bcda/client/bluebutton.go

```diff
@@ -256,3 +259,13 @@ func HashIdentifier(toHash string) (hashedValue string) {
 	}
 	return hex.EncodeToString(pbkdf2.Key([]byte(toHash), pepper, blueButtonIter, 32, sha256.New))
 }
+
+func UpdateParamWithLastUpdated(params *url.Values, since string, jobCreation time.Time) {
+	// only set the lower bound parameter if it exists and begins with "ge" (to align with what is expected in _lastUpdated)
+	if len(since) > 0 && strings.HasPrefix(since, "ge") {
+		params.Set("_lastUpdated", since)
+
+		// now add the upper bound based on job creation
+		params.Add("_lastUpdated", "lt"+jobCreation.Format(time.RFC3339Nano))
+	}
```

Comment:
Same type of comment on the upper bound. The upper bound should be a `le`.

Reply:
Updated.